### PR TITLE
Add Servers - Documentation page that includes details on the Version…

### PR DESCRIPTION
**Pull request by odscjames**
… and Language Switcher

https://github.com/open-contracting/infrastructure/issues/68

### CODE REVIEW

**Comment by jpmckinney** [comment]
Can you integrate:

* the first part into https://ocds-standard-development-handbook.readthedocs.io/en/latest/systems/servers/
* second part into https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/technical/deployment/#update-the-deployment-repository

? Thanks!

**Comment by odscjames** [comment]
Updated after discussion with @duncandewhurst

File: docs/infrastructure/technical/deployment.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Deployment
```
Comment (jpmckinney) [inline docs/infrastructure/technical/deployment.md]: Although OC4IDS is not an OCDS profile, it seems a bit much to create three new pages to add a bit of content about how OC4IDS gets deployed. I think I'd prefer a `profiles/infrastructure.md` page – which can certainly start with an admonition that OC4IDS isn't a profile, if we think that's needed – that follows a format similar to `profiles/ppp.md` to describe the exceptions/nuances here.

**Comment by odscjames** [comment]
Moved as requested.

**Pull request merged**